doc/examples/reconst_fwdti.py

```diff
@@ -0,0 +1,185 @@
+"""
+==========================================================================
+Using the free water elimination model to remove free water contamination
+==========================================================================
+
+As shown previously (see :ref:`example_reconst_dti`), the diffusion tensor
+model is a simple way to characterize the diffusion anisotropy. However, this
```

Comment:
"characterize the diffusion anisotropy" => "characterize diffusion anisotropy" or "characterize the anisotropy of diffusion"